<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ScanProgressBase</class>
  <widget class="QDialog" name="ScanProgressBase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>550</width>
        <height>151</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Scan Progress</string>
    </property>
    <widget class="QWidget" name="Layout7" >
      <property name="geometry" >
        <rect>
          <x>0</x>
          <y>0</y>
          <width>550</width>
          <height>151</height>
        </rect>
      </property>
      <layout class="QVBoxLayout" >
        <property name="margin" >
          <number>0</number>
        </property>
        <property name="spacing" >
          <number>6</number>
        </property>
        <item>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="QLabel" name="TextLabel1" >
                <property name="minimumSize" >
                  <size>
                    <width>150</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="text" >
                  <string>Current directory:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="fDirectory" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>1</vsizetype>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>-</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="QLabel" name="TextLabel3" >
                <property name="minimumSize" >
                  <size>
                    <width>150</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="text" >
                  <string>Current file:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="fFile" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>1</vsizetype>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>-</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="QLabel" name="TextLabel1_2" >
                <property name="minimumSize" >
                  <size>
                    <width>150</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="text" >
                  <string>Files scanned:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="fFilesScanned" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>1</vsizetype>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>0</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="QLabel" name="TextLabel5" >
                <property name="minimumSize" >
                  <size>
                    <width>150</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="text" >
                  <string>Directories scanned:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="fDirsScanned" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>1</vsizetype>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>0</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="QLabel" name="TextLabel7" >
                <property name="minimumSize" >
                  <size>
                    <width>150</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="text" >
                  <string>Directories left:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLabel" name="fDirsLeft" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>1</vsizetype>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>0</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
  </widget>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
